packages/dap/src/hpkeConfig.ts

```diff
@@ -43,10 +43,18 @@ export class HpkeConfig implements Encodable {
     if (id !== Math.floor(id) || id < 0 || id > 255) {
       throw new Error("id must be an integer in [0, 255]");
     }
-
-    this.validate(KemId, "kemId");
-    this.validate(KdfId, "kdfId");
-    this.validate(AeadId, "aeadId");
+    if (kemId !== Math.floor(kemId) || kemId < 0 || kemId > 0xffff) {
```

Comment:
In practice, how would any of these scenarios come about? The wire encoding ensures that we get a u16, and the constructor is only ever called from decode

Reply:
The `HpkeConfig` constructor is publicly accessible, since the class is exported. I think it's unlikely that it'll be used directly, but maybe someone will come along and re-implement key distribution.